### ISSUE TITLE
graphviz on mac not reliably re-reading document changes

### ISSUE THREAD

**Issue by mgrubb** [ellson/graphviz#28]
When editing a graph dot file in an external editor then saving the changes, Graphviz.app doesn't alway (almost never) re-reads the changed file and update the graph accordingly.  Also when adding a new edge the graph typically completely disappears and leaves a dark gray screen that has to be closed, and the graph file re-opened.  Sometimes when adding an edge it doesn't do this though.  When syntax errors are present in the graph file, it won't re-read the contents after they have been corrected.
I'm running this on the latest version of Mac OS X Yosemite (10.10) on the newest 15" Macbook Pro with retina display (mid 2014 model)

**Comment by tewe** [comment]
Same issue on 10.10.3 with the 2.36.0 package. Even saving without changing the inode (most editor do) isn't noticed by the GUI. And there's no manual refresh.

**Comment by magneticnorth** [comment]
I tried with homebrew and it seems OK (OSX 10.4.4) on a trivial example. You might try this too.

Maybe someone who is very familiar with OSX programming can look into this?  I think the code was originally written by Glen Low (glen.low@pixelglow.com).   With increasing distractions for the people that wrote most of Graphviz, you might be better off with a commercial application or one in Java (that is non-native and thus avoids the ever-changing UI toolkit problems).  Hope this helps.

Stephen North

**Comment by tewe** [comment]
Homebrew did fix the problem. Maybe it just needs to be built against a current SDK. So a new official binary would be nice.

**Comment by graphviz** [comment]
> On Aug 4, 2015, at 11:14 AM, tewe <notifications@github.com> wrote:
> 
> Homebrew did fix the problem. Maybe it just needs to be built against a current SDK. So a new official binary would be nice.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ellson/graphviz/issues/28#issuecomment-127645821>.
> 
Yes.  Would someone like to donate a Mac Mini and we’ll put it in John Ellson’s data center :-)
